integration_tests/citus-cdc/create_source.sql

```diff
@@ -20,5 +20,30 @@ CREATE TABLE orders_rw (
     schema.name = 'public',
     table.name = 'orders',
     slot.name = 'orders_dbz_slot',
-    publication.create.enable = 'true'
```

Comment:
so `publication.create.enable = 'true'` is a default behavior?

Reply:
Yes, user can create and provide their own pub.